release_tester/arangodb/sh.py

```diff
@@ -244,6 +244,31 @@ def hotbackup_check_for_nonbackup_data(self):
 
         return res
 
+    @step
+    def hotbackup_wait_for_ready_after_restore(self):
+        """check whether the data is in there or not."""
+        logging.info("running version check")
+        #  || db.this_collection_will_not_be_backed_up._length() != 0
+        # // do we care?
+        js_script_string = """
+        if (!arango.isConnected()) {
+          throw new Error('connecting the database failed');
+        }
+        let timeout = 15;
+        while (db._databases().length == 0)  {
+          if (timeout == 0) {		
+            throw new Error("Databases array is still empty after 15s!");		
+          }		
+          require("internal").sleep(1);		
+          timeout -= 1;
+        }
+        """
+        logging.debug("script to be executed: " + str(js_script_string))
```

Comment:
Why we need to put entire script in output? Can we just print message that we are waiting for database's array?

Reply:
I had the issue of too much magic happening a while back, and hence printed it everywhere.